.github/renovate.json

```diff
@@ -44,7 +48,12 @@
       "ignoreUnstable": false,
       "followTag": "latest",
       "reviewers": ["@jolg42", "@williamluke4", "millsp"],
-      "packageNames": ["@prisma/engines", "@prisma/engines-version", "@prisma/fetch-engine", "@prisma/get-platform"]
+      "packageNames": [
```

Comment:
Why did this already list these 4?

Reply:
Because I added them in the original PR but I did not add them to the exclusions